api/delete.py

```diff
@@ -16,8 +17,15 @@ async def delete(payload: RequestPayload):
         encoder=encoder,
         dimensions=payload.encoder.dimensions,
     )
+    summary_vector_service: BaseVectorDatabase = get_vector_service(
+        index_name=f"{payload.index_name}{SUMMARY_SUFFIX}",
+        credentials=payload.vector_database,
+        encoder=encoder,
+        dimensions=payload.encoder.dimensions,
+    )
 
     for file in payload.files:
         data = await vector_service.delete(file_url=file.url)
+        await summary_vector_service.delete(file_url=file.url)
```

Comment:
@homanp Previously we were not deleting the summary index. I think it makes sense to remove it, is there any reason why we shouldn't?